hw_23.09.23/task2.py

```diff
@@ -0,0 +1,51 @@
+import random
+
+
+class Boss:
+    def __init__(self, name: str, company: str):
+        self.id = random.randint(1, 999)
+        self.name = name
+        self.company = company
+        self._workers = []
+
+    @property
+    def workers(self):
+        text = f'Workers for {self.name} are:\n'
+        for employee in self._workers:
+            text += f'{employee["id_"]}, {employee["name"]}\n'
+
+        return text
+
+    @workers.setter
+    def workers(self, worker):
+        self._workers.append(worker)
+
+    def __repr__(self):
+        return f"Boss: {self.id} {self.name}"
+
+
+class Worker:
+    def __init__(self, name: str, boss: Boss):
+        self.id_ = random.randint(1, 999)
```

Comment:
There is a chance you'll run into a collision here

Reply:
It's better to generate id in a predictable manner (try to avoid `random` in your programs)